### PR TITLE
DynamoRevit LinkElementNodes

### PR DESCRIPTION
**Pull request by dariaivanciucova**
### Purpose

This PR contains new nodes for Dynamo Revit - all falling under LinkElement category.
image
image

- LinkElement.GetGeometry
- LinkElement.GetLocation
- LinkElement.BoundingBox
- LinkElement.LinkInverseTransform
- LinkElement.LinkTransform
- LinkElement.ByRayBounce

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] Snapshot of UI changes, if any.

### Reviewers

@Amoursol
@hwahlstrom

### FYIs

@dnenov

### CODE REVIEW

File: src/Libraries/RevitNodesUI/RevitTypes.cs [hunk 1]
```diff
@@ -197,7 +197,7 @@ public DirectShapeRoomBoundingOption(IEnumerable<PortModel> inPorts, IEnumerable
     [OutPortTypes("string")]
     public class DetailLevel : CustomGenericEnumerationDropDown
     {
-        private const string outputName = "Detail Level";
```
Comment (reddyashish) [inline src/Libraries/RevitNodesUI/RevitTypes.cs:200]: Why are we changing the node output name to lowercamel case here?

File: src/Libraries/RevitNodes/Application/Document.cs [hunk 2]
```diff
@@ -325,6 +325,7 @@ public DynamoUnits.Unit UnitTypeBySpecType(ForgeType specType)
 
             return Unit.ByTypeID(cleanUnitTypeId.TypeId);
         }
+
```
Comment (reddyashish) [inline src/Libraries/RevitNodes/Application/Document.cs:328]: No changes to this file.

File: src/Libraries/RevitNodesUI/RevitTypes.cs (hunk 1 above)
Comment (dariaivanciucova) [reply src/Libraries/RevitNodesUI/RevitTypes.cs:200]: @reddyashish this was to match the names of the output ports provided by Helena
image
https://www.figma.com/file/4JFNaoH1YdbEh9tXQ9JRB6/DynamoRevit-nodes?type=design&node-id=0-1&t=adSKSuTxnt8rxXcg-0

File: src/DynamoRevitIcons/DynamoRevitIcons.csproj [hunk 3]
```diff
@@ -34,7 +34,9 @@
   </PropertyGroup>
   <ItemGroup>
     <EmbeddedResource Include="DSRevitNodesUIImages.resx" />
-    <EmbeddedResource Include="RevitNodesImages.resx" />
+    <EmbeddedResource Include="RevitNodesImages.resx">
```
Comment (reddyashish) [inline src/DynamoRevitIcons/DynamoRevitIcons.csproj:37]: Is this not needed in the new PR? The number of files changed seem to be different for the 2 PR's. Can you please cross check that.

**Comment by QilongTang** [comment]
@reddyashish Thank you Reddy, I will do it for you

**Pull request closed without merging**